boilerplate/boilerplate.py

```diff
@@ -33,12 +33,12 @@
     nargs='*')
 
 # Rootdir defaults to the directory **above** the repo-infra dir.
-rootdir = os.path.dirname(__file__) + "./../../../"
+rootdir = os.path.dirname(__file__) + "./../../"
 rootdir = os.path.abspath(rootdir)
 parser.add_argument(
     "--rootdir", default=rootdir, help="root directory to examine")
 
-default_boilerplate_dir = os.path.join(rootdir, "csi-driver-nfs/hack/boilerplate")
+default_boilerplate_dir = os.path.join(rootdir, "hack/boilerplate")
```

Comment:
This is broken. If `rootdir` is the directory above "release-tools", then there is no "hack/boilerplate" directory in it.

I think `os.path.abspath(os.path.dirname(__file__))` is the right default?

Reply:
sorry, `hack/boilerplate` is my project directory.
yes, `default_boilerplate_dir=os.path.abspath(os.path.dirname(__file__))` is right